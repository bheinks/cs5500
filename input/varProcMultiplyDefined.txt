(*
	Procedure name is multiply defined (same as variable in 
      same scope)
*)

program p;

var proc : integer;

procedure proc;
begin
  write(99)
end;

begin
  write(5)
end.

