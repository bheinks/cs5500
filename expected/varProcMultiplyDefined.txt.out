

>>> Entering new scope...

+++ Adding p to symbol table with type PROGRAM

+++ Adding proc to symbol table with type INTEGER

+++ Adding proc to symbol table with type PROCEDURE
Line 10: Multiply defined identifier
